misc/extension_api_validation/4.2-stable.expected

```diff
@@ -35,7 +35,7 @@ GH-84976
 Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/FinalAction/values/FINAL_ACTION_CONTINUE': value changed value in new API, from 2.0 to 0.
 Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/FinalAction/values/FINAL_ACTION_MAX': value changed value in new API, from 3.0 to 2.
 Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/InitialAction/values/INITIAL_ACTION_CLEAR': value changed value in new API, from 0.0 to 1.
-Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/InitialAction/values/INITIAL_ACTION_CLEAR_REGION_CONTINUE': value changed value in new API, from 2.0 to 0.
+Validate extension JSON: Error: Field 'classes/RenderingDevice/enums/InitialAction/values/INITIAL_ACTION_CLEAR_REGION_CONTINUE': value changed value in new API, from 2.0 to 1.
```

Comment:
You need to add a new entry for this not edit an old one 🙂

Reply:
This is a changelog between 4.2-stable and current `master`, so it shouldn't include iterations on the API within the dev cycle. I.e. changing this in place seems correct.

Reply:
Yes, @akien-mga's reasoning is pretty much why I did it in place, as it's just a fix for an incorrect mapping that the PR introduced.